gettsim/visualization.py

```diff
@@ -571,3 +331,246 @@ def _kth_order_successors(dag, node, order):
     if 1 <= order:
         for successor in dag.successors(node):
             yield from _kth_order_successors(dag, successor, order=order - 1)
+
+
+def plot_dag(
+    functions,
+    source_code=False,
+    targets=None,
+    columns_overriding_functions=None,
+    check_minimal_specification="ignore",
+    selectors=None,
+    orientation="v",
+    showlabels=True,
+    showlegend=False,
+):
+    """Plot the dag of the tax and transfer system.
+
+    Parameters
+    ----------
+    functions : str, pathlib.Path, callable, module, imports statements, dict
+        Functions can be anything of the specified types and a list of the same objects.
+        If the object is a dictionary, the keys of the dictionary are used as a name
+        instead of the function name. For all other objects, the name is inferred from
+        the function name.
+    source_code str,
+        Experimental feature which makes the source code of the functions or tbe node
```

Comment:
List the potential values of the argument here. I would rather call it `hover_info`. Please also move the argument to the end of the argument list (not the second position)

Reply:
I just saw that you currently just implemented it as a boolean variable which doesn't fit the description here.

Maybe boolean makes sense if we show labels on hover-mode whenever `source_code = False`?

Or what was your idea?